platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/system/transaction/StateSignatureTransaction.java

```diff
@@ -142,50 +74,43 @@ public int getSize() {
      */
     @Override
     public void serialize(final SerializableDataOutputStream out) throws IOException {
```

Comment:
Because this is something that will influence the hash of an event, we need to make sure we always write this in the same byte format as when we read it. This means we will have to support both the old and new serialization format until migration is complete. Not as hard as it sounds, we have done this sort of thing before. Ping me if you'd like to discuss.

Reply:
The serialization code has changed, but the format remains the same, so the hash remains the same. This class will be deleted once we change the serialization and hashing of events, at that point, we will need a migration. Until then, everything is backwards compatible.

Reply:
Makes sense.